src/tax-lot/tax-lot.service.spec.ts

```diff
@@ -60,7 +60,7 @@ describe("TaxLotController", () => {
   });
 
   describe("findZoningDistrictByTaxLotBbl", () => {
-    it("should return an array of zoning district(s) when requesting a valid bbl", async () => {
+    it.only("should return an array of zoning district(s) when requesting a valid bbl", async () => {
```

Comment:
Let's remove this `only` (I think we added it when we were testing out the tax lot relational mocks)

Reply:
oops